Class-Code/Elementary-CA/pryan/sketch.js

```diff
@@ -1,14 +1,60 @@
-//Your Name(s)
-//Project Name
-//Date
-
-/* A brief description of what the program is */
+//Pat Ryan
+//Elementary CA
+//Feb 21 2019
 
+/An Elementary Cellular Automata/
 
+let gensize = 61
+let indexVal = Math.floor(gensize/2);
+const ruleset =[0,1,0,1,1,0,1,0];
+let gen1 = Array(gensize).fill(0);
+ gen1[indexVal]=1;
+let ca =[];
+let gridsize;
 function setup() {
-    createCanvas(400, 400);
+    createCanvas(800, 400);
+    gridsize = width/gen1.length
+    console.log(gridsize)
+    ca.push(gen1)
 }
 
 function draw() {
-    background(220);
+    background(255);
```

Comment:
Generally check your formatting, indentation, etc. The easy way to do this is with the VS extension Beautify